implementations/java-jsonschema-validator/bowtieJsonSchemaValidator.java

```diff
@@ -0,0 +1,215 @@
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.ValidationMessage;
+import java.io.*;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.jar.Attributes;
+import java.util.jar.Manifest;
+
+public class BowtieJsonSchemaValidator {
```

Comment:
The name of the file should be capitalized (pascal case). You even refer to it with capitalized name in gradle script - I don't even know how it finds this file, honestly 😄

Reply:
That's strange! It keeps automatically changing to camelCase. Is there a git command I might have executed incorrectly? I only did one reset, and since then, it has been automatically converting to camelCase.

Reply:
Are you on macOS?

Reply:
Windows

Reply:
Windows